### PR TITLE
Update Summary Budget Table

### LINKED ISSUE #2290: Update Summary Budget Table
update summary budget table as follows (before and after screenshots from design task #2247):

Before:

After:

### This task is done when...
 - [ ] Change header "totals" to have Medicaid Total Computable the right most table column
 - [ ] Remove the term "Project" from this page".
 - [ ] Replace Contracted Resources with Private Contractor.
 - [ ] Replace Non-Personnel with Other State Expenses.
 - [ ] In the instructions change Medicaid Share = Medicaid Total Computable
 - [ ] In the table change Medicaid Total = Medicaid Total Computable

### PR DESCRIPTION
**Pull request by pphillips-fearless**
Update the summary budget table based on the screenshot.  Closes #2290 

### This pull request changes...

- Change header "totals" to have Medicaid Total Computable the right most table column
- Remove the term "Project" from this page".
- Replace Contracted Resources with Private Contractor.
- Replace Non-Personnel with Other State Expenses.
- In the instructions change Medicaid Share = Medicaid Total Computable
- In the table change Medicaid Total = Medicaid Total Computable
- sort by State Staff, Other State Expenses, and Private Contractor

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
~~- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented~~
~~- [ ] The change has been documented~~
~~- [ ] Associated OpenAPI documentation has been updated~~
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by pphillips-fearless** [comment]
the table column widths are a little different, so let me know if they're OK or not.

**Comment by beparticular** [comment]
@jeromeleecms do you see any issue with the last column being a little wider? does that give any indication that the column is something that it isn't, or give it too much weight? 


Screen Shot 2020-06-17 at 11 46 40 AM

**Comment by jeromeleecms** [comment]
@beparticular I don't have an issue with the last column be wider as long as the content is visible for State, Federal, and Medicaid Total Computable. 

From our parking lot discussion, this change in column size is due to the rendering of the CSS with no fix width, so I'm not particularly keen on applying fixed widths unless there is a specific use case for it.

File: web/src/containers/BudgetSummary.js [hunk 1]
```diff
@@ -42,16 +45,18 @@ DataRow.propTypes = {
...
-        category={key}
-        data={data[key][year]}
-        entries={entries}
-        title={categoryLookup[key]}
-        year={year}
-      />
-    ))}
+    {Object.keys(data)
+      .sort(categorySort)
+      .map(key => (
```
Comment (mgwalker) [inline web/src/containers/BudgetSummary.js:50]: Would it make sense to do the same thing here as my comment in #2293, where the `categoryLookup` is pre-sorted so it doesn't have to be done on each instance of the `DataRowGroup`?

E.g.,

```js
const categories = Object.keys(categoryLookup).sort(categorySort);
```

And then here it'd be:

```js
categories.map(key => (...
```

Comment (pphillips-fearless) [reply web/src/containers/BudgetSummary.js:50]: yes, agreed!

**Pull request merged**